src/main/java/thePackmaster/patches/shamanpack/FineTuneLineWidthPatch.java

```diff
@@ -0,0 +1,53 @@
+package thePackmaster.patches.shamanpack;
+
+import com.evacipated.cardcrawl.modthespire.lib.*;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DescriptionLine;
+import javassist.CannotCompileException;
+import javassist.CtBehavior;
+import javassist.expr.ExprEditor;
+import javassist.expr.NewExpr;
+import thePackmaster.cards.shamanpack.FreezeAndBurn;
+import thePackmaster.cards.shamanpack.FueledByEmbers;
+
+@SpirePatch(clz = AbstractCard.class, method = "initializeDescription")
+public class FineTuneLineWidthPatch {
+    public static final float WIDTH_ADJUSTMENT = 20.0f;
+    public static boolean addWidthBack = false;
+
+    private static boolean matches(AbstractCard card, String word) {
```

Comment:
Why does this patch exist, you ask? Here is exhibit A.

Without the patch, absolutely horrific line breaks:
![card without patch](https://user-images.githubusercontent.com/62083413/210256625-b2b9c828-aaee-4035-a4ae-9e21ddcaa7c7.png)

With the patch, bliss:
![card with patch](https://user-images.githubusercontent.com/62083413/210256646-a3639ca1-c6a4-4e78-bbee-96f7f51c207a.png)

Note that this is restricted to handle only specific cards, so there's no risk of it affecting other cards or other packs. I've done something similar in my other mods without any issues.

Reply:
Oh, and the other card that needing fine-tuning was Freeze and Burn, which had "enemies" on its own line as the final line of the card. The classic poor line break for "Deal N damage to ALL enemies."